scripts/pwned-passwords.sh

```diff
@@ -5,45 +5,31 @@ set -eu
 submit_to_s3='false'
 pwned_directory="pwned_passwords"
 number_of_passwords=3000000
-pwned_url="https://downloads.pwnedpasswords.com/passwords/pwned-passwords-sha1-ordered-by-count-v8.7z"
-pwned_7z="${pwned_directory}/pwned-passwords.7z"
-pwned_file="${pwned_directory}/pwned-passwords.txt"
+pwned_file="${pwned_directory}/pwned_passwords.txt"
+pwned_file_unsorted="${pwned_directory}/pwned_passwords_unsorted.txt"
```

Comment:
this is "unsorted" alphabetically but sorted by frequency right? maybe we could add that for clarity?
```suggestion
pwned_file_unsorted_alphabetically="${pwned_directory}/pwned_passwords_unsorted_alphabetically.txt"
```

Reply:
That's a good point, though I think it's most strictly accurate to say that the file contains the top X password hashes, but there is no guaranteed sort within the set of those top passwords ([PairingHeap iterator](https://github.com/18F/identity-idp/blob/206bbc6a829162fda6c16482671e56fe44fcb063/app/javascript/packages/pwned-passwords/pairing-heap.js#L31-L51) walks the data structure, but not in a way that guarantees order). Maybe some variation of "top hashes" could work here.

Reply:
yeah top x hashes is definitely clearer

Reply:
> this is "unsorted" alphabetically but sorted by frequency right? maybe we could add that for clarity?

Renamed to "pwned_passwords_top_hashes.txt" in 697d6c313.